arelle/plugin/transforms/SEC/conf/tests.xml

```diff
@@ -399,6 +401,8 @@
       <variation input="Q3'1999" output="1999-09-30" result="valid" />
       <variation input="1999, Q5" output="" result="invalid" />
       <variation input="1999, Q4" output="1999-12-31" result="valid" />
+      <variation input="1999, Q44" output="1999-12-31" result="invalid" />
+      <variation input="1999, 44" output="1999-12-31" result="invalid" />

```

Comment:
Rerun `extractTestcase.sh` and check in those changes?  It seems like we should either delete that file or keep it in sync with `tests.xml`.

Reply:
done c77450fe